lib/publify_textfilter_ma_says.rb

```diff
@@ -6,22 +6,22 @@ class MaSays< TextFilterPlugin::MacroPost
 
       def self.help_text
         %{
-You can use `<publify:masays>` to display a Ma Says snippet.  Example:
+You can use the following code to display a Ma Says snippet. Example:
 
-    <publify:masays url="https://www.moneyadviceservice.org.uk/en/campaigns/saving-for-a-holiday">
-      Saving for a Holiday
-    </publify:masays>}
+<hr class="snippet" data-snippet-name="masays" data-snippet-text="Saving for a Holiday" data-snippet-url="https://www.moneyadviceservice.org.uk/en/campaigns/saving-for-a-holiday" />
+}
       end
 
-      def self.macrofilter(_, _, attrib, _, text)
-        url = attrib['url']
+      def self.macrofilter(_, _, attrib, _)
+        url = attrib['data-snippet-url']
+        text = attrib['data-snippet-text']
 
         "<div class=\"snippet next-steps\">
            <div class=\"next-steps__heading\">
              <span>MA</span> says
            </div>
 
-           <a data-ck href=\"#{url}\" class=\"next-steps__container-link\">
+           <a href=\"#{url}\" class=\"next-steps__container-link\">
```

Comment:
As discussed this is the code that is actually output to the article page, instead of the <hr />